databases_preview.go

```diff
@@ -26,9 +27,17 @@ var _ DatabasesPreviewService = (*PreviewClient)(nil)
 
 // DatabaseNG stands for Database Next Generation.
 type DatabaseNG struct {
-	App      App       `json:"app"`
-	Addon    Addon     `json:"addon"`
-	Database *Database `json:"database,omitempty"`
+	ID         string    `json:"id"`
+	Name       string    `json:"name"`
+	ProjectID  string    `json:"project_id"`
+	Technology string    `json:"technology"`
+	Plan       string    `json:"plan"`
+	Database   *Database `json:"database,omitempty"`
+	App        App       `json:"app"`
+}
+
+type databaseApiResponse struct {
+	Database DatabaseNG `json:"database"`
```

Comment:
question: Does this really work? Here you are listing, but you are not using a slice. Shouldn't it be:
```suggestion
	[]Database DatabaseNG `json:"database"`
```
Actually I understand better now by looking at the code below. But shouldn't our API standard be:
```json
{
	"databases": [
		{
			"id": "database"	
		},
	]
}
```